clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/converters/ResizeAsgAtomicOperationConverter.groovy

```diff
@@ -38,4 +40,36 @@ class ResizeAsgAtomicOperationConverter extends AbstractAtomicOperationsCredenti
     converted.credentials = getCredentialsObject(input.credentials as String)
     converted
   }
+
+  /**
+   * Converts legacy resize asg descriptions to their modern equivalent.
+   *
+   * Expects to be run after {@see AsgNameToServerGroupNameDescriptionPreProcessor}.
+   */
+  @Order(0)
+  @Component
+  static class ResizeAsgDescriptionPreProcessor implements AtomicOperationDescriptionPreProcessor {
+    @Override
+    boolean supports(Class descriptionClass, Map description) {
+      return descriptionClass == ResizeAsgDescription
+    }
+
+    @Override
+    Map process(Map description) {
+      if (description.asgs) {
+        return description
+      }
+
+      description.asgs = description.regions.collect {
```

Comment:
we should stick a  metric on any Preprocessor that ends up manipulating a payload so we have an indication of whether they are still necessary or not (Maybe that bubbles up to whichever is calling the preprocessors to compare the before and after as that maybe has more useful context to tag the metric with)